cpp/tests/transform/row_bit_count_test.cu

```diff
@@ -453,6 +453,67 @@ TEST_F(RowBitCount, NestedTypes)
   }
 }
 
+TEST_F(RowBitCount, NullsInStringsList)
+{
+  using namespace cudf;
+  using namespace cudf::test;
```

Comment:
This seem unnecessary, as I see `cudf::` and `cudf::test` are used everywhere in this file.

Reply:
That's a good point. I meant to remove the explicit `cudf::`.